src/stories/Typography.stories.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/vue';
+import { withKnobs, select, number } from '@storybook/addon-knobs';
+import { component as VueCodeHighlight } from 'vue-code-highlight';
+import './styles/Typography.scss';
+
+storiesOf('Typography/Typographies', module)
```

Comment:
Please, don't forget to use the new Storybook syntax =D

You can find more information on this PR:
- https://github.com/homeday-de/homeday-blocks/pull/512

Or by looking at some components that already are using the new format:
- https://github.com/homeday-de/homeday-blocks/blob/develop/src/stories/HdButton.stories.js
- https://github.com/homeday-de/homeday-blocks/blob/develop/src/stories/HdDashedList.stories.js
- https://github.com/homeday-de/homeday-blocks/blob/develop/src/stories/HdNotification.stories.js

If you need any help, just ping me 😄

Reply:
Done!